soc/nordic/nrf54h/power.c

```diff
@@ -9,12 +9,16 @@
 #include <zephyr/pm/policy.h>
 #include <zephyr/arch/common/pm_s2ram.h>
 #include <hal/nrf_resetinfo.h>
-#include <hal/nrf_lrcconf.h>
 #include <hal/nrf_memconf.h>
 #include <zephyr/cache.h>
 #include <power.h>
+#include <soc_lrcconf.h>
 #include "pm_s2ram.h"
 
+#if !IS_ENABLED(CONFIG_SOC_NRF54H20_CPURAD)
```

Comment:
```suggestion
#if !defined(CONFIG_SOC_NRF54H20_CPURAD)
```
`IS_ENABLED` should only be used in C expressions. See: https://github.com/zephyrproject-rtos/zephyr/blob/7e892f8c6fd67b678ae5a6f06a73f878e9c2377a/include/zephyr/sys/util_macro.h#L117-L118